chaoscenter/graphql/server/manifests/namespace/1b_argo_deployment.yaml

```diff
@@ -31,6 +31,8 @@ spec:
 apiVersion: apps/v1
 kind: Deployment
 metadata:
+  labels:
+    module: execution-plane
```

Comment:
Can we add these changes in cluster mode as well here: https://github.com/amityt/litmus/blob/manifest-labels/chaoscenter/graphql/server/manifests/cluster/1c_argo_deployment.yaml